src/types/fn_type.rs

```diff
@@ -9,7 +9,7 @@ use crate::AddressSpace;
 use crate::context::ContextRef;
 use crate::support::LLVMString;
 use crate::types::traits::AsTypeRef;
-use crate::types::{PointerType, Type, BasicTypeEnum};
+use crate::types::{AnyType, PointerType, Type, BasicTypeEnum};
```

Comment:
Is this import necessary here?

Reply:
Yes, `impl fmt::Debug` uses self.print_to_strng()